### ISSUE TITLE
publish at F-Droid appstore

### ISSUE THREAD

**Issue by noerw** [CodeforKarlsruhe/opensense#11]
It would be a great addition for non-playstore users to have the app in [F-Droid](https://f-droid.org).
At a quick glance it seems that the app fulfills the [inclusion policy](https://f-droid.org/wiki/page/Inclusion_Policy).

see https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/

**Comment by dbaelz** [comment]
Thanks für the good idea.

I'm not quit sure if the app fulfills the inclusion police due the mapbox and android support library dependencies. In the next days I'll take a deeper look at the policy. When possible, I'll publish the app on F-Droid.

**Comment by dbaelz** [comment]
Finally, I found the time to look at it. In my opionen the app could be publish it to f-droid. There are same pros and cons to do so

Pros:
- More possible user (maybe...)
- Support open source even more (maybe, see first point of cons)

Cons:
- The app contains libraries which are  "antifeatures" for F-Droid (see [Antifeature:NonFreeDep](https://f-droid.org/wiki/page/Antifeature:NonFreeDep)). In my opinion it's questionable if there is an advantage to install the app via f-droid, because it requires proprietary components like Google Play Sevices and therefore doesn't work on a clean AOSP/Custom Rom image.
- The upload process is significantly more complicated and elaborate than I expected. "Install a bunch of libraries/runtimes, start the virtual machine, execute a bunch of scripts, commit a bunch of stuff and build the whole f-droid project" isn't that enjoyable or a way to get me motivated to support f-droid. To say it that kindly...

In conclusion I postponed the upload of 0.2.0. I'll probably do it for a later version (rather > 0.3.0) with more time and a working pipeline. The issue stays open so I keep it in mind.

**Comment by silberzwiebel** [comment]
I'm running the self-compiled app on a tablet without any google stuff. So, I guess your first point reduces to adding the antifeature flag only (is it the map-provider library that is not free?).
For me, it is always disadvantageous to install apps from google play, because I need to find some ways to grab the actual apk and no automatical updates are provided (although things are a lot easier with Yalp Store around).

I might help with the actual process of adding the app to F-Droid.

**Comment by JuliDi** [comment]
Since the app is no longer available on the Play Store, it would be awesome if you could publish it via f-droid.
